packages/docusaurus-plugin-content-blog/src/types.ts

```diff
@@ -96,6 +96,7 @@ export type PluginOptions = RemarkAndRehypePluginOptions & {
   admonitions: Record<string, unknown>;
   authorsMapPath: string;
   readingTime: ReadingTimeFunctionOption;
+  postSortDirection: string;
```

Comment:
```suggestion
  postSortDirection: 'asc' | 'desc';
```

Use union type to constrain to only two values.